index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- NAVBAR HERE -->
+    <nav>
+      <ul>
+        <li class="logo-left">
+          <img src="img/03.png" alt="" srcset="">
+          <p>Original Trombones</p>
+        </li>
```

Comment:
- I think it would be better if you could align your logo and the text " original trombones" 
Please change the paragraph tag to a heading tag in line 17.